core/components/ui/select/select.tsx

```diff
@@ -60,11 +60,11 @@ const Select = forwardRef<ElementRef<typeof SelectPrimitive.Trigger>, Props>(
             </SelectPrimitive.ScrollUpButton>
             <SelectPrimitive.Viewport className="h-[var(--radix-select-trigger-height)] w-full min-w-[var(--radix-select-trigger-width)]">
               {options.map((option) => {
-                const { value, label, ...optionProps } = option;
+                const { value, label, key, ...optionProps } = option;
 
                 return (
                   <SelectPrimitive.Item
-                    key={`${id}-${value}`}
+                    key={key ?? `${id}-${value}`}
```

Comment:
I guess we got to do this for component that iterate.

Reply:
Yeah unfortunately :(